source/documentation/partials/_IE022.md

```diff
@@ -1,4 +1,7 @@
 ## IE022 Notification To Amend Declaration
+
+**Note:** This message type is out of scope until after November 2023.
+
```

Comment:
Looks OK. 

Presumably, we want to say "out of scope" instead of "not applicable".